libraries/botbuilder-azure-blobs/tests/blobsStorage.test.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+
+const assert = require('assert');
+const sinon = require('sinon');
+const { BlobsStorage } = require('../lib');
+
+const connectionString = process.env.AZURE_BLOB_STORAGE_CONNECTION_STRING;
+const containerName = process.env.AZURE_BLOB_STORAGE_CONTAINER;
+
+const maybeClient = () =>
```

Comment:
This construct, while handy in dev, means lots of tests will be skipped in CI. Do we have a mechanism to provide an emulated or test storage environment in CI?